core/promise/consumer.go

```diff
@@ -18,10 +18,31 @@
 package promise
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	ethereum "github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/mysteriumnetwork/node/communication"
+	"github.com/mysteriumnetwork/node/core/promise/storage"
+	"github.com/mysteriumnetwork/node/identity"
+	"github.com/mysteriumnetwork/node/service_discovery/dto"
 )
 
-type consumer struct{}
+var (
+	errLowAmount          = fmt.Errorf("promise amount less than the service proposal price")
```

Comment:
Why not errors.New()? You dont need formatting in this case.

Reply:
Well, both `errors.New()` and `fmt.Errorf()` doing the same under the hood. 
It's more style question. Do we have any recommendation which one to use in the project?

Reply:
Most of the time we used errors.New(), sorta indicates that we are creating "New" message.

Reply:
ok, clear, changed it.